collector/processors/odigosroutingfilterprocessor/config.go

```diff
@@ -0,0 +1,23 @@
+package odigosroutingfilterprocessor
+
+import (
+	"go.opentelemetry.io/collector/component"
+)
+
+type Config struct {
+	MatchConditions []string `mapstructure:"match_conditions"`
+	MatchMap        map[string]struct{}
```

Comment:
this is not part of the config. I think it should go directly as a member of the `filterProcessor`